### DISCUSSION TITLE
Error: Property 'replaceAll' does not exist on type 'string'

### DISCUSSION THREAD

**Discussion by BeyersNel** [codewars/codewars.com#2826] [Q&A] [answered]
```
TSError: ⨯ Unable to compile TypeScript:
solution.ts:3:40 - error TS2550: Property 'replaceAll' does not exist on type 'string'. Do you need to change your target library? Try changing the `lib` compiler option to 'esnext' or later.
```

Any solution to this?

**Comment by Raulcarvalhomelo** [comment, marked as answer]
You may solve the problem using RegExp and global flag. The global flag is what makes replace run on all occurrences.

"1399/06/08".replace(/\//g, "_") // "1399_06_08"

https://www.thecodeteacher.com/question/80066/angular---Property-'replaceAll'-does-not-exist-on-type-'string'
